### PR TITLE
Update doc with pci pf support

### PR DESCRIPTION
**Pull request by adrianchiris**
SRIOV network device plugin supports discovery and allocation of non SR-IOV network devices.
Align documentation with this use-case.

In addition, fix typo in CONTRIBUTING doc (separate commit in this PR)

### CODE REVIEW

**Comment by adrianchiris** [comment]
@Eoghan1232 @zshi-redhat  PTAL

File: README.md [hunk 1]
```diff
@@ -48,11 +49,13 @@ To deploy workloads with SR-IOV VF this plugin needs to work together with the f
 
   - Retrieves allocated network device information of a Pod
 
-- SR-IOV CNI
-
-  - During Pod creation, plumbs allocated SR-IOV VF to a Pods network namespace using VF information given by the meta plugin
-
-  - On Pod deletion, reset and release the VF from the Pod
+- A CNI capable of consuming the network device allocated to the Pod
```
Comment (zshi-redhat) [inline README.md:52]: Maybe remove the `SR-IOV VF` in line 46?
or modify it to `To deploy workloads with SR-IOV VF or PCI PF, this plugin needs to ...`

Comment (Eoghan1232) [reply README.md:52]: +1

Comment (Eoghan1232) [reply README.md:52]: otherwise LGTM.

**Review by zshi-redhat — APPROVED** [review]

**Review by Eoghan1232 — APPROVED** [review]

**Pull request merged**